src/app/find-help/accommodation-search/accommodation-search-form/accommodation-search-form.component.ts

```diff
@@ -43,4 +63,17 @@ export class AccommodationSearchFormComponent implements OnInit {
     await this.storeUrlService.setCustomPaginatorParam(param);
     this.search.emit(this.data);
   }
+
+  async clearFilters(): Promise<void> {
+    this.data = {};
+    const { page, size, sort } = this.route.snapshot.queryParams;
+    const param: Params = {
+      page,
+      size,
+      sort,
+      ...cleanForm,
+    };
+    await this.storeUrlService.setCustomPaginatorParam(param);
+    this.search.emit(this.data);
```

Comment:
po kliknięciu "wyczyść" strzelamy do API tak jak byśmy kliknęli "Wyszukaj". Na pewno tak chcemy?

Reply:
Moim zdaniem powinniśmy odświeżyć też listę wyników, więc strzał do api jest uzasadniony.

Reply:
Zastanowiłbym się w takim razie tylko nad literałem przycisku, może bardziej `Wyczyść wyniki` albo coś takiego. Ale to nic pilnego oczywiście :)